src/containers/ReportArticle/state/reducer.test.js

```diff
@@ -0,0 +1,93 @@
+import {
+  FETCH_REPORT_TYPES, FETCH_REPORT_TYPES_SUCCESS,
+  REPORTING_SUCCESS, REPORTING_FAILURE,
+  REPORT_MORE_THAN_ONCE, FETCH_REPORT_TYPES_FAILURE,
+} from './types';
+import { fetchTypesResponse, successResponse, errorResp } from './mock';
+import reducer from './reducer';
+
+
+const initialState = {
+  reportTypes: {},
+  isReporting: false,
+  errors: "",
+  message: "",
+  success: false,
+  hasError: false,
+};
+
+const action = {
+  payload: {},
+};
+
+describe('<ArticleReport /> reducer ', () => {
```

Comment:
Refactor this test suite title

Reply:
I have refactored it to `The Article Report` as advised. Thanks